_docs/_latest/configure/notifier/index.md

```diff
@@ -234,24 +234,18 @@ the UI or API:
 - Note: You will need to have either owner, editor or service management roles
   in the Project in order to enable the API
 
-1. Enable the  API connector config to Cloud SCC.  Specifically, this means
-in the Forseti project server bucket, edit the `configs/forseti_conf_server.yaml`,
- as follows:
+1. Enable the  API connector config to Cloud SCC by configuring the following fields in 
+your Terraform configuration:
 
-- Open `forseti-security/configs/forseti_conf_server.yaml` from the GCS bucket.
-- Navigate to the `notifier` > `violation` > `cscc` section.
-
-  The options below are available for you to configure:
-
-  * `enabled:`
+  * `cscc_violations_enabled`
     * **Description**: Whether to send notification to Cloud SCC.
-    * **Valid values**: one of valid `true` or `false`
+    * **Valid values**: One of valid `true` or `false`
   
-  * `source_id`
+  * `cscc_source_id`
     * **Description**: ID from the Cloud SCC beta on-boarding. **This must be added**
     to use the Beta API integration.
```

Comment:
Can you please remove any mention of `Beta` for CSCC, while you are here?  CSCC is GA now.  Please look at lines 227, 230 245, 246, and 250

Reply:
Fixed!